MetaMask/metamask-extension
# Show a notification about BSC Swaps if a user is on the BSC Mainnet

## Explanation
Show a notification about BSC Swaps if a user is on the BSC Mainnet. 

When a user clicks on "Start swapping" or close the "What's New" window, we will not show the notification again.

## Manual testing steps 
- Uninstall the MM browser extension
- Install and set it up again
- In the What's New window you should see 3 notifications during initial start
- Add BSC Mainnet as a custom network: https://academy.binance.com/en/articles/connecting-metamask-to-binance-smart-chain
- Make sure the BSC Mainnet is selected
- Close the MM extension
- Open it again
- A single notification about BSC Swaps should appear when you are on BSC Mainnet
- Click on "Start swapping"
- The Build Quote page is displayed
- Close the extension
- Open it again.. no notifications should be displayed
- Switch to Ethereum Mainnet
- It should show no notification as well

## Screenshots
A user is on the BSC Mainnet:
<img width="354" alt="image" src="https://user-images.githubusercontent.com/80175477/118163457-4ab20300-b422-11eb-97e9-e150046bdb7b.png">

After clicking on the "Start swapping" button: 
<img width="366" alt="image" src="https://user-images.githubusercontent.com/80175477/118129657-bc775600-b3fc-11eb-9e73-3827ff3d97ce.png">

No new notification is displayed after reopening the extension:
<img width="370" alt="image" src="https://user-images.githubusercontent.com/80175477/118129697-cdc06280-b3fc-11eb-88b4-3c87aebf8b70.png">
